be/src/service/staros_worker.cpp

```diff
@@ -103,12 +103,9 @@ absl::StatusOr<std::shared_ptr<fslib::FileSystem>> StarOSWorker::get_shard_files
         fslib::Configuration localconf;
         ShardInfo& info = shard_iter->second.shard_info;
 
-        // properties["cache_enabled"] = "true"
-        auto iter_use_cache = info.properties.find("cache_enabled");
         // FIXME: currently the cache root dir is set from be.conf, could be changed in future
         std::string cache_dir = config::starlet_cache_dir;
-        bool cache_enabled = !cache_dir.empty() && iter_use_cache != info.properties.end() &&
-                             iter_use_cache->second.compare("true") == 0;
+        auto cache_setting = info.get_cache_setting();
```

Comment:
this part should be reserved, in case cache_dir is not set, cache will be disabled too even shard is enabled.

Reply:
ok